src/project/roots.rs

```diff
@@ -19,7 +19,7 @@ pub struct Roots {
 
 /// A path to a gc root.
 #[derive(Hash, PartialEq, Eq, Clone, Debug, Serialize, Deserialize)]
-pub struct RootPath(PathBuf);
+pub struct RootPath(pub PathBuf);
```

Comment:
Not super elegant IMO, but I think `RootPath` should just go away at some point, so I don't mind breaking the abstraction here.